conda-forge.yml

```diff
@@ -1,14 +1,14 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
-  osx_arm64: osx_64
+    linux_64: linux_64
+    osx_64: osx_64
+    osx_arm64: osx_64
```

Comment:
@stewartboogert why this? can't we natively compile on arm? (see osx build failure)

Reply:
Ok, I see now that you already gave the native build and then cross compilation a go. I don't know how we can get such an "bad cpu type" error if we compile natively on ARM

Reply:
We are still waiting (yes, even now) for native `osx-arm64` runners in azure pipelines. So currently cross-compilation is the only option for `osx-arm64` builds.